docs/source/_templates/layout.html

```diff
@@ -1,16 +1,53 @@
 {% extends "!layout.html" %}
 
+
+
 <link rel="canonical" href="{{ theme_canonical_url }}{{ pagename }}.html" />
+
 {% block menu %}
 <div>
   <a style="color:#F05732" href="{{ theme_canonical_url }}{{ pagename }}.html">
     You are viewing unstable developer preview docs.
     Click here to view docs for latest stable release.
   </a>
 </div>
+
 {{ super() }}
 {% endblock %}
 
+{% block sidebartitle %}
```

Comment:
This definitely deserves comment that it is mostly copied from https://github.com/rtfd/sphinx_rtd_theme/blob/master/sphinx_rtd_theme/layout.html#L92-L122 with modifications only to the url.

Reply:
This whole layout was copied from that theme, originally. Much of it was removed, to simplify.